benchmarks/dicodile_text.py

```diff
@@ -1,24 +1,22 @@
-import os
-import pathlib
+from datetime import datetime
 import numpy as np
+import os
 import pandas as pd
+import pathlib
 from scipy import signal
-from datetime import datetime
 from scipy.optimize import linear_sum_assignment
 
-from dicodile.utils import check_random_state
-
 # Import for online DL
 import spams
 from sklearn.feature_extraction.image import extract_patches_2d
 
 # Import for CDL
 from dicodile import dicodile
-
+from dicodile.data.home import DATA_HOME
```

Comment:
I would such constants in a config.py file rather than home.py

wdyt @tomMoral ?

Reply:
Seems like a nice idea to move everything to `dicodile.config.py`.

Reply:
Shall we do it in another pull request, or in this one? It might affect other examples and benchmarks?